### PR TITLE
add test for cloning empty color

### PR DESCRIPTION
**Pull request by dliv**
Shows issue 67 is already fixed (by commit 5b625b3abacb514cabb8f77395da71259a4aa348).

That hash is for the commit that adds immutability. Adding this test should prevent a regression (although I assume cloning might be removed at some point since it seems pointless with an immutable API).

### CODE REVIEW

**Comment by MoOx** [comment]
> I assume cloning might be removed at some point

Make sense.

**Comment by Qix-** [comment]
It will. :+1:

File: test/index.js [hunk 1]
```diff
@@ -713,6 +713,23 @@ it('Clone', function () {
 	deepEqual(clone.rgbaArray(), [10, 20, 30, 1]);
 });
 
+it('Clone: default constructor', function () {
+	var defaultColor = Color();
+	var clonedFromDefault = defaultColor.clone();
```
Comment (Qix-) [inline test/index.js]: Could add a `notEqual` test for these two, just to be complete.

Comment (dliv) [inline test/index.js]: So: 

```
	// changes to clone don't affect original
	notEqual(clonedFromDefault.rgb(100, 100, 100), defaultColor);
```

or 

```
	// clone is not same instance
	notStrictEqual(defaultColor, clonedFromDefault);
```

or something else?

Comment (Qix-) [inline test/index.js]: The second.

**Comment by Qix-** [comment]
Thanks! :+1: :dancer:
